### PR TITLE
Update ping.markdown

### PR DESCRIPTION
**Pull request by Azimoth**
## Proposed change

Added a more explicit description regarding the naming of hosts under presence detection. 

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [X] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

### CODE REVIEW

File: source/_integrations/ping.markdown [hunk 1]
```diff
@@ -89,7 +89,7 @@ device_tracker:
 
 {% configuration %}
 hosts:
-  description: List of device names and their corresponding IP address or hostname.
+  description: List of device names and their corresponding IP address or hostname. (Lower case, numbers and underscore only, see [entity names](/configuration/troubleshooting/#entity-names))
```
Comment (DubhAd) [inline source/_integrations/ping.markdown]: I'd suggest something like:

```
List of device names and their corresponding IP address or hostname. Device names must conform to the standard requirements of lower case, numbers and underscore only - see [entity names](/configuration/troubleshooting/#entity-names).
```

Comment (Azimoth) [reply source/_integrations/ping.markdown]: Sure, fine with me.

Comment (Azimoth) [reply source/_integrations/ping.markdown]: Do I need to... nevermind, figured it out :)

**Review by frenck — APPROVED** [review]
Thanks, @Azimoth! 👍

**Pull request merged**